roles/download/defaults/main.yml

```diff
@@ -448,8 +448,8 @@ haproxy_image_tag: 2.3
 # Coredns version should be supported by corefile-migration (or at least work with)
 # bundle with kubeadm; if not 'basic' upgrade can sometimes fail
 coredns_version: "1.7.0"
```

Comment:
only `coredns_image_tag` use `coredns_version`, is it more appropriate to add here? 🤔️